src/pages/Home/Home.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { nanoid } from '@reduxjs/toolkit';
+import Slider from '../../components/Slider/Slider';
+import { ContextHeading } from '../../globalStyles.styled';
+import {
+  OverviewWrapper,
+  OverviewPara,
+  OverviewSpan,
+  CategoriesWrapper,
+} from './Home.styled';
+import Category from '../../components/Category/Category';
+
+const Home = () => {
+  const tickets = useSelector((state) => state.tickets);
+  if (Object.keys(tickets).length === 0) {
+    return <></>;
+  }
+  const { page, categoryImages } = tickets;
+  const altText = [
+    {
+      id: nanoid(),
+      type: 'pages',
+      value: page.totalPages,
+    },
+    {
+      id: nanoid(),
+      type: 'events',
+      value: page.totalElements,
+    },
+    {
+      id: nanoid(),
+      type: 'categories',
+      value: page.categories,
+    },
+  ];
+
+  return (
+    <>
+      <Slider />
+      <ContextHeading>Overview</ContextHeading>
+      <OverviewWrapper>
+        {
+          altText.map((each) => (
+            <OverviewPara key={each.id}>
+              {
+                Number(each.value).toLocaleString()
+              }
+              <OverviewSpan>{each.type}</OverviewSpan>
+            </OverviewPara>
+          ))
+        }
+      </OverviewWrapper>
```

Comment:
The Home page should have a search bar that could be used as a filter using given parameters, e.g. category name. 👍🏻

Reply:
@zainsadaqat actually the API structure is too heavy and because of some Object coming with Array have not enough Data. So that's why I already filtered the data so that it will be easy for user to work on.